monitorConfig/production.ts

```diff
@@ -50,5 +50,26 @@ export const config: Config = {
 				},
 			],
 		},
+		{
+			networkId: 10, // ZKEVM
```

Comment:
10 is optimism mainnet network id

Reply:
@aminlatifi Thanks, fixed and replaced with `1101`(and `2442` for staging)